clang/lib/Sema/SemaExprMember.cpp

```diff
@@ -1287,7 +1287,10 @@ static ExprResult LookupMemberExpr(Sema &S, LookupResult &R,
     return ExprError();
 
   QualType BaseType = BaseExpr.get()->getType();
+
+#if 0
```

Comment:
Hmm?

Reply:
This assertion should be deleted since the lookup context could be the current instantiation (ergo dependent).